pasos-para-investigar.md

```diff
@@ -0,0 +1,51 @@
+## Pasos para Investigar
+
```

Comment:
Since this is the title and start of the document, it would be better to use an `h1`. Also, it would be nice to have a short introductory description of the document that explains its use.